astropy/analytic_functions/tests/test_blackbody.py

```diff
@@ -32,7 +32,7 @@ def test_blackbody_scipy():
 
     """
     flux_unit = u.Watt / (u.m ** 2 * u.um)
-    wave = np.logspace(0, 8, 1e6) * u.AA
+    wave = np.logspace(0, 8, 100000) * u.AA
```

Comment:
thanks for pinging me - no I can't remember any specific reason behind the number when I first wrote the code. If it can be sped up, that seems sensible.